.github/workflows/solana-v2-rust.yml

```diff
@@ -144,12 +144,12 @@ jobs:
       - name: Install
         run: yarn install --ignore-scripts --frozen-lockfile
 
-      - name: Lint
-        run: yarn lint
-
       - name: Build program
         run: yarn build
 
+      - name: Lint
```

Comment:
I think this was changed for a reason? Or was it breaking?

Reply:
Hmm I don't remember changing that so maybe it switched during a backmerge. Will look